### DISCUSSION TITLE
Handle `default` (not found) page

### DISCUSSION THREAD

**Discussion by warloque** [react-static/react-static#1441] [General]
The problem is react-static wouldn't find **routeInfo.json** file corresponding to the random URL which is not amongst the predefined routes. 

This simple router layout is supposed to handle any `wrong` uri
`
<Router>
   <Routes path="*" />
   <NotFound default />
</Router>
`
and it indeed works (as well as with render props method). 
But that 404 error for the **routeInfo.json** in the console is pretty dissappointing. 
_Any thoughts on how to quietly handle 404 cases?_

**Comment by SleeplessByte** [comment]
We won't know something doesn't exist until we try it; or at least that's how I think the current code works. Happy to change the behaviour.

Keeping a list of all pages that exist is _not_ an option.
